netlogo-gui/src/main/lab/gui/ProtocolEditable.scala

```diff
@@ -28,7 +28,9 @@ class ProtocolEditable(protocol: LabProtocol,
 
   val propertySet = {
     import scala.collection.JavaConverters._
-    List(Property("hint", Property.Label, I18N.gui("hint")),
+    List(Property("hint", Property.Label, "<html>"+I18N.gui("hint")+"</html>",
+                  backgroundColor = new java.awt.Color(128, 200, 128, 64),
```

Comment:
could add comment indicating what color this is e.g. "light green" - although it might have a better name